### PR TITLE
Functions have become actions again

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

We no longer want to rename actions to functions. So revert that name change.
The new work was done behind a feature flag so this doesn't affect existing behaviour.

## QA steps

Deploy a charm with actions and have juju-v3 feature flag set

juju run myunit/0 myaction
juju list-actions
juju show-action myaction

## Documentation changes

Discourse will need updating.

### CODE REVIEW

File: cmd/juju/action/run_test.go [hunk 1]
```diff
@@ -79,113 +79,113 @@ func (s *CallSuite) TestInit(c *gc.C) {
...
-		expectError: "invalid unit or function name \"something-strange-\"",
+		args:        []string{invalidUnitId, validUnitId, "valid-action-name"},
+		expectError: "invalid unit or action name \"something-strange-\"",
 	}, {
 		should:         "work with multiple valid units",
-		args:           []string{validUnitId, validUnitId2, "valid-function-name"},
+		args:           []string{validUnitId, validUnitId2, "valid-action-name"},
 		expectUnits:    []string{validUnitId, validUnitId2},
-		expectFunction: "valid-function-name",
+		expectFunction: "valid-action-name",
```
Comment (howbazaar) [inline cmd/juju/action/run_test.go]: Should this be expectAction?

File: cmd/juju/action/run_test.go [hunk 2]
```diff
@@ -255,80 +255,80 @@ func (s *CallSuite) TestCall(c *gc.C) {
...
 		withFunctionResults: []params.ActionResult{{
 			Action: &params.Action{Tag: validActionTagString},
 			Error:  common.ServerError(errors.New("database error")),
 		}},
 		expectedErr: "database error",
 	}, {
 		should:   "fail with invalid tag in result",
-		withArgs: []string{validUnitId, "some-function"},
+		withArgs: []string{validUnitId, "some-action"},
 		withFunctionResults: []params.ActionResult{{
```
Comment (howbazaar) [inline cmd/juju/action/run_test.go]: withActionResults

File: cmd/juju/action/show.go [hunk 3]
```diff
@@ -96,11 +89,7 @@ func (c *showCommand) Run(ctx *cmd.Context) error {
 	}
 	info, ok := actions[c.functionName]
```
Comment (howbazaar) [inline cmd/juju/action/show.go]: actionName

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**